src/components/modal/styles.ts

```diff
@@ -23,40 +29,49 @@ export const ElModal = styled.div`
   top: 50%;
   left: 50%;
   transform: translate(-50%, -50%);
-  box-shadow: 0px 4px 16px 0px rgba(34, 43, 51, 0.16);
-  border-radius: var(--default-border-radius);
   background: white;
   z-index: 99;
-  width: 65%;
-  min-width: 300px;
-  max-width: 800px;
-  max-height: 80vh;
   overflow: auto;
-  padding: 1.25rem 1.5rem;
+  box-shadow: var(--elevation-xl, 0px 4px 16px 0px rgba(96, 120, 144, 0.16));
+  border-radius: var(--corner-lg, 8px);
+  padding: var(--spacing-5, 20px) var(--spacing-6, 24px);
 
   &.${elIsActive} {
     display: block;
     position: fixed;
   }
+
+  /* default modal size settings */
+  min-width: 300px;
+  max-height: 100%;
+  width: calc(100% - var(--unit-8, 32px));
+  max-width: 30rem;
+
+  /* small modal size settings */
+  &.${elModalSmallVariant} {
+    width: calc(100% - var(--unit-16, 64px));
+    max-width: 20rem;
+  }
 `
 
 export const ElModalHeader = styled.div`
```

Comment:
is there any maximum lines for modal (dialog) title that have been discussed before? with current styling we aren't set the max lines or truncating the title

Reply:
raise this question to Design Team, https://www.figma.com/design/J6EneNGfpk2GcUuymiOKW4?node-id=164-4207&m=dev#978863499

Reply:
> No truncation on any heading, thank you. It can be as many lines as is needed (but is almost always 1-2)

From Karmen in the conversation above